src/routes/courses.ts

```diff
@@ -1,8 +1,17 @@
-const express = require('express');
+import {
+  Router,
+  type Request,
+  type Response,
+  type NextFunction
+} from 'express';
 
-const router = express.Router();
+import { COURSES } from '../data/courses';
 
-const courses = require('../data/courses');
+import { type Course } from '../models/course';
```

Comment:
Using the word 'type' in imports is optional and it doesn't affect the import statement's functionality or type usage.